Formula/lib/libdex.rb

```diff
@@ -22,8 +22,21 @@ class Libdex < Formula
   depends_on "vala" => :build # for vapigen
   depends_on "glib"
 
+  on_macos do
+    depends_on "gcc"
+  end
+
+  fails_with :clang do
+    cause "Requires libatomic"
+  end
+
   def install
```

Comment:
```suggestion
  def install
    ENV.append_to_cflags "-L#{Formula["gcc"].opt_lib}/gcc/current" if OS.mac?
```
Might work. If not, maybe adding
```ruby
ENV.append "LDFLAGS", "-L#{Formula["gcc"].opt_lib}/gcc/current" if OS.mac?
```
will help.